lib/test/mock_loader.js

```diff
@@ -0,0 +1,228 @@
+'use strict';
+const {
+  SafeMap,
+  globalThis,
+} = primordials;
+const {
+  Atomics: {
+    notify: AtomicsNotify,
+    store: AtomicsStore,
+  },
+} = globalThis;
```

Comment:
nit: if you destructure `globalThis` from the `primordials` object, the linter will check all the keys are ASCII-ordered.
```suggestion
  globalThis: {
    Atomics: {
      notify: AtomicsNotify,
      store: AtomicsStore,
    },
  },
} = primordials;
```